mage_ai/frontend/pages/datasets/[slug].tsx

```diff
@@ -361,10 +362,17 @@ function Data() {
       <Spacing mt={UNIT} />
       {headEl}
       <Spacing mt={2} />
-      <SuggestionsList
-        featureSet={datasetResponse}
-        featureSetId={slug}
-      />
+      <ApiReloader
```

Comment:
You'll need to call the `setApiReloads` method in the success callback of your `useUpdate` request, and also wrap that useUpdate call with `useMutation` from the `react-query` package, which isn't in the repo yet.